src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -212,15 +223,35 @@ public async Task<List<UserViewModel>> ListPreReleaseUsersAsync()
                         return ValidationActionResult(UserDoesNotExist);
                     }
 
-                    var userReleaseRoles = await GetUserReleaseRoles(user.Id);
+                    var userReleaseRoles = _contentDbContext.UserReleaseRoles
+                        .Include(urr => urr.Release)
+                        .ThenInclude(r => r.Publication)
+                        .ToList()
+                        .Where(urr => urr.UserId == Guid.Parse(userId) &&
+                                      IsLatestVersionOfRelease(urr.Release.Publication.Releases, urr.Release.Id))
+                        .Select(x => new UserReleaseRoleViewModel
+                        {
+                            Id = x.Id,
+                            Publication = _contentDbContext.Publications
+                                .Where(p => p.Releases.Any(r => r.Id == x.ReleaseId))
+                                .Select(p => new IdTitlePair {Id = p.Id, Title = p.Title}).FirstOrDefault(),
+                            Release = _contentDbContext.Releases
+                                .Where(r => r.Id == x.ReleaseId)
+                                .Select(r => new IdTitlePair {Id = r.Id, Title = r.Title}).FirstOrDefault(),
+                            ReleaseRole = new EnumExtensions.EnumValue {Name = x.Role.GetEnumLabel(), Value = 0}
+                        })
+                        .ToList()
+                        .OrderBy(x => x.Publication.Title)
+                        .ThenBy(x => x.Release.Title)
+                        .ToList();
```

Comment:
I think you could use `ToListAsync` here

Reply:
Seems not:
```UserManagementService.cs(246, 26): [CS1061] 'IOrderedEnumerable<UserReleaseRoleViewModel>' does not contain a definition for 'ToListAsync' and no accessible extension method 'ToListAsync' accepting a first argument of type 'IOrderedEnumerable<UserReleaseRoleViewModel>' could be found (are you missing a using directive or an assembly reference?)```